src/client/hooks/useAuthentication.js

```diff
@@ -32,6 +34,10 @@ export function useAuthentication() {
       if (user) {
         setIsAuthenticated(true);
         setIsLoading(false);
+
+        const uid = user.uid;
```

Comment:
Could be written like this but it is up to you :) 
```suggestion
        const { uid } = user;
```

Reply:
i will, Tq niels